mesonbuild/mdist.py

```diff
@@ -136,11 +212,19 @@ def git_root(self, dir_: str) -> Path:
 
     def have_dirty_index(self) -> bool:
         '''Check whether there are uncommitted changes in git'''
-        ret = subprocess.call(['git', '-C', self.src_root, 'diff-index', '--quiet', 'HEAD'])
+        ret = subprocess.call(
+            ['git', '-C', self.src_root, 'diff-index', '--quiet', 'HEAD']
+        )
         return ret == 1
 
-    def copy_git(self, src: T.Union[str, os.PathLike], distdir: str, revision: str = 'HEAD',
-                 prefix: T.Optional[str] = None, subdir: T.Optional[str] = None) -> None:
+    def create_source_dir_from_repo(
```

Comment:
Rename it from "copy_git" to "copy_repo" instead, and skip reindenting the function signature.